src/components/form/HdGoogleAutocomplete.vue

```diff
@@ -249,7 +253,15 @@ export default {
       if (this.required && this.isEmpty) {
         this.showError(this.t.FORM.VALIDATION.REQUIRED);
       } else {
```

Comment:
What about the non-required and empty fields, probably we don't want to run the customValidators on them, do you think we should add this condition to each customValidator or just handle that case in a generic way:

```js
if (this.required && this.isEmpty) {
  // Show is required error
} else if (!this.isEmpty) {
  // Check the custom validators
} else {
  // just hide the errors
}
```

Reply:
Good catch. I think handling it generically is the way to go. I will add some logic